Error while installing the latest version 10.2.2 from source

I am running into the following error when running the `make` on the new version of Citus. Did anyone run into this?
Here are the steps which I followed:
```
-bash-4.1$ PG_CONFIG=/var/illumio_pce/external/bin/pg_config ./configure --without-libcurl --without-lz4 --without-zstd
checking for a sed that does not truncate output... /bin/sed
checking for gawk... gawk
checking for flex... /usr/bin/flex
configure: building against PostgreSQL 13
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking size of void *... 8
checking whether gcc supports -std=gnu99... yes
checking whether gcc supports -Wall... yes
checking whether gcc supports -Wextra... yes
checking whether gcc supports -Wno-unused-parameter... yes
checking whether gcc supports -Wno-sign-compare... yes
checking whether gcc supports -Wno-missing-field-initializers... yes
checking whether gcc supports -Wno-clobbered... yes
checking whether gcc supports -Wno-gnu-variable-sized-type-not-at-end... no
checking whether gcc supports -Wno-declaration-after-statement... yes
checking whether gcc supports -Wendif-labels... yes
checking whether gcc supports -Wmissing-format-attribute... yes
checking whether gcc supports -Wmissing-declarations... yes
checking whether gcc supports -Wmissing-prototypes... yes
checking whether gcc supports -Wshadow... yes
checking whether gcc supports -Werror=vla... yes
checking whether gcc supports -Werror=implicit-int... yes
checking whether gcc supports -Werror=implicit-function-declaration... yes
checking whether gcc supports -Werror=return-type... yes
checking whether gcc supports -fstack-clash-protection... no
checking for git... /usr/bin/git
checking for .git... no
configure: creating ./config.status
config.status: creating Makefile.global
config.status: creating src/include/citus_config.h
config.status: src/include/citus_config.h is unchanged
config.status: creating src/include/citus_version.h
config.status: src/include/citus_version.h is unchanged
```

```
-bash-4.1$ make
Makefile:51: warning: overriding commands for target `check'
/var/illumio_pce/external/lib/postgresql/pgxs/src/makefiles/pgxs.mk:433: warning: ignoring old commands for target `check'
make -C src/backend/distributed/ all
make[1]: Entering directory `/home/vagrant/build_system/SOURCES/citus-10.2.2/src/backend/distributed'
gcc -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Werror=vla -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -I/tmp/volatile/deps/external/include -I/usr/include -fPIC -std=gnu99 -Wall -Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing-field-initializers -Wno-clobbered -Wno-declaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wmissing-declarations -Wmissing-prototypes -Wshadow -Werror=vla -Werror=implicit-int -Werror=implicit-function-declaration -Werror=return-type -I '/home/vagrant/build_system/SOURCES/citus-10.2.2/src/include' -I'../../../src/include' -I. -I./ -I/var/illumio_pce/external/include/postgresql/server -I/var/illumio_pce/external/include/postgresql/internal  -D_GNU_SOURCE  -I/var/illumio_pce/external/include -I/home/vagrant/build_system/SOURCES/citus-10.2.2/vendor/safestringlib/include  -c -o ../columnar/columnar_tableam.o ../columnar/columnar_tableam.c
../columnar/columnar_tableam.c:88: error: redefinition of typedef ‘ColumnarScanDesc’
/home/vagrant/build_system/SOURCES/citus-10.2.2/src/include/columnar/columnar_tableam.h:48: note: previous declaration of ‘ColumnarScanDesc’ was here
../columnar/columnar_tableam.c: In function ‘columnar_beginscan_extended’:
../columnar/columnar_tableam.c:231: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘row_number_to_tid’:
../columnar/columnar_tableam.c:353: warning: missing braces around initializer
../columnar/columnar_tableam.c:353: warning: (near initialization for ‘tid.ip_blkid’)
../columnar/columnar_tableam.c: In function ‘ErrorIfInvalidRowNumber’:
../columnar/columnar_tableam.c:385: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:390: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_parallelscan_estimate’:
../columnar/columnar_tableam.c:402: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_parallelscan_initialize’:
../columnar/columnar_tableam.c:409: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_parallelscan_reinitialize’:
../columnar/columnar_tableam.c:416: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_index_fetch_begin’:
../columnar/columnar_tableam.c:429: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_fetch_row_version’:
../columnar/columnar_tableam.c:625: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_get_latest_tid’:
../columnar/columnar_tableam.c:633: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_tid_valid’:
../columnar/columnar_tableam.c:640: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_compute_xid_horizon_for_tuples’:
../columnar/columnar_tableam.c:711: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_insert_speculative’:
../columnar/columnar_tableam.c:754: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_complete_speculative’:
../columnar/columnar_tableam.c:762: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_delete’:
../columnar/columnar_tableam.c:806: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_update’:
../columnar/columnar_tableam.c:816: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_tuple_lock’:
../columnar/columnar_tableam.c:826: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_relation_set_new_filenode’:
../columnar/columnar_tableam.c:850: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_relation_copy_data’:
../columnar/columnar_tableam.c:910: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_relation_copy_for_cluster’:
../columnar/columnar_tableam.c:938: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘LogRelationStats’:
../columnar/columnar_tableam.c:1172: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘TruncateColumnar’:
../columnar/columnar_tableam.c:1214: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:1249: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_index_build_range_scan’:
../columnar/columnar_tableam.c:1355: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:1364: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_scan_sample_next_block’:
../columnar/columnar_tableam.c:1826: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘columnar_scan_sample_next_tuple’:
../columnar/columnar_tableam.c:1834: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘ColumnarTriggerCreateHook’:
../columnar/columnar_tableam.c:2039: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘ColumnarProcessUtility’:
../columnar/columnar_tableam.c:2107: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘ColumnarCheckLogicalReplication’:
../columnar/columnar_tableam.c:2295: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘alter_columnar_table_set’:
../columnar/columnar_tableam.c:2448: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2457: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2467: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2473: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2484: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2490: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2502: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2505: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2516: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2522: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘alter_columnar_table_reset’:
../columnar/columnar_tableam.c:2577: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2586: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2593: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2601: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2610: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c:2618: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘upgrade_columnar_storage’:
../columnar/columnar_tableam.c:2669: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
../columnar/columnar_tableam.c: In function ‘downgrade_columnar_storage’:
../columnar/columnar_tableam.c:2705: warning: declaration of ‘__errno_location’ shadows a global declaration
/usr/include/bits/errno.h:47: warning: shadowed declaration is here
make[1]: *** [../columnar/columnar_tableam.o] Error 1
make[1]: Leaving directory `/home/vagrant/build_system/SOURCES/citus-10.2.2/src/backend/distributed'
make: *** [extension] Error 2
```
Please let me know if I am missing anything. Thanks!

--- a/src/backend/columnar/columnar_tableam.c
+++ b/src/backend/columnar/columnar_tableam.c
@@ -85,7 +85,6 @@ typedef struct ColumnarScanDescData
 	List *scanQual;
 } ColumnarScanDescData;
 
-typedef struct ColumnarScanDescData *ColumnarScanDesc;
 
 /*
  * IndexFetchColumnarData is the scan state passed between index_fetch_begin,

